### ISSUE TITLE
Rebrand the app to Atomatic

### ISSUE THREAD

**Issue by adeekshith** [adeekshith/watomatic#433]
Replace "Watomatic" to "Atomatic" wherever it is visible to the user but keep the package name and other internal representation as Watomatic for now.

This includes the
- [X] app
- [x] website: This branch: https://github.com/adeekshith/watomatic/tree/docs/ghpages
- ~play store listing~

Context: https://techhub.social/@dsoft/112275160163391428

**Comment by geekiyer** [comment]
I see you have self-assigned it to yourself. But, I can try to take this up incase you are short of time.

**Comment by adeekshith** [comment]
Hi @geekiyer 👋 That would be great! Thank you! 😃
Feel free to ask if you have any questions

**Comment by geekiyer** [comment]
Thanks @adeekshith! I have an initial PR #435 to rename a few things. How can I help renaming the assets/media/screenshots/website/etc?

**Comment by adeekshith** [comment]
Thank you @geekiyer !!

Website code is in the same repo in branch `docs/ghpages`. You can fork it from that branch and make a PR. Branch here: https://github.com/adeekshith/watomatic/tree/docs/ghpages 

For screenshots, we may have to redo them. Only if you have time, can you make screenshots with device frame like this: https://github.com/adeekshith/watomatic/blob/main/media/screenshots/wato-1-8-light-main-pixel3a.png
Exclude the words "Whatsapp" in the screenshots and for now lets also exclude whatsapp chat screenshots. I forgot the app I used for device screenshots but any app works. Also if device screenshots are difficult, just plain screenshots should also work

**Comment by geekiyer** [comment]
Thanks! I am working on it!

**Comment by adeekshith** [comment]
I was thinking something generic like:

- "Atomatic - Auto reply for messaging apps
- Helps you send auto replies for your favorite messaging apps

Keeping it generic like this is also better because we want to support more messaging apps soon.

**Comment by adeekshith** [comment]
Hi @geekiyer 👋, were you able to make any progress on this? Let me know if you need any help :)
Thanks!

**Comment by geekiyer** [comment]
> Hi @geekiyer 👋, were you able to make any progress on this? Let me know if you need any help :) Thanks!

@adeekshith Yes. I have the new assets almost ready! Bit busy with year end gatherings and activities. Will open a PR in a couple of days.

**Comment by geekiyer** [comment]
@adeekshith Please take a look at #437. Thanks

**Comment by adeekshith** [comment]
Hi @geekiyer Happy new year and thank you so much! Merged the PR just now. That is great help 😃

**Comment by geekiyer** [comment]
@adeekshith Happy New Year! Here is PR #438 for the website.
